### PR TITLE
Add ImmobilienScout24.de

### PR DESCRIPTION
**Pull request by IDKwhattoputhere**

### CODE REVIEW

**Comment by J0WI** [comment]
Please also include other subdomains. We have a guide about how to find them:
https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md

**Comment by IDKwhattoputhere** [comment]
That's only how the rule should look like or am I missing something? I can't seem to find where it says how to find the subdomains.

**Comment by J0WI** [comment]
```
If you're not sure what subdomains might exist, you can iteratively use google queries and enumerate the list of results like such:

    site:*.eff.org
    site:*.eff.org -site:www.eff.org
    site:*.eff.org -site:www.eff.org -site:ssd.eff.org

... and so on.
```
Please also document non working subdomains for future maintenance.

**Comment by IDKwhattoputhere** [comment]
Not sure how I missed that. Anyway I made a change to the file. Any mistakes?

File: src/chrome/content/rules/ImmobilienScout24.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	<target host="tracking.immobilienscout24.de" />
+	
+	<!-- not working:
+		blog.immobilienscout24.de
+		developer.immobilienscout24.de
+		forum.immobilienscout24.de
+		www.focus.immobilienscout24.de
+	-->
+
+	<rule from="^http://(?:www\.)?immobilienscout24\.de/"
```
Comment (J0WI) [inline src/chrome/content/rules/ImmobilienScout24.xml]: Please remove the `?:` and also add a comment for the mismatch on ^immobilienscout24.de
Comments should be on the top above the ruleset.

**Comment by J0WI** [comment]
Thanks a lot!
I found an additional host that should be included here:
www.static-immobilienscout24.de (^ does not exist)
wwworig.static-immobilienscout24.de (mismatch)

**Comment by IDKwhattoputhere** [comment]
I feel like that last change shouldn't cause an error. What now?

**Comment by J0WI** [comment]
Travis hang some times, tests should pass if I would rerun them. But there are still a few points:
`www.static-immobilienscout24.de` itself can be safely rewritten (add it as target host). The comment for non-existent ^ is correct, but you should rephrase it (doesn't fit in the non-woking hosts list).

The comments should be realy on the top, means above the ruleset tag. It would be useful to have a small description why the non working ones are not working. You can also make categories.
See [other rulesets](https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/cesky-hosting.cz.xml) for an example.

Just ping me if you have questions.

**Comment by IDKwhattoputhere** [comment]
pong

**Comment by J0WI** [comment]
Looks great now, thanks for contributing!

**Pull request merged**